xAPI.md

```diff
@@ -1536,8 +1547,9 @@ The table below defines the Score Object.
 	</tr>
 </table>
 
-The properties of the score object are based on the corresponding properties of cmi.score as defined in SCORM 2004 
-4th Edition. 
+The properties of the Score Object are based on the corresponding properties of cmi.score as defined in SCORM 2004 
+4th Edition. If both "scaled" and "raw" properties are provided, the exact relationship between their two values is determined 
```

Comment:
In an attempt to avoid a lowercase should I reworded this heavily:

``` The "scaled" and "raw" properties do not necessarily relate directly as scaling and normalization can
be applied differently by Activity Providers within different Communities of Practice. Scaling and normalization are 
outside the scope of this specification. ```